ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -71,17 +73,21 @@ const AssetListItem = ({
             updateSendToken({
               address: tokenAddress,
               decimals: tokenDecimals,
-              symbol: tokenSymbol,
+              symbol: primaryIsFiat ? secondarySymbol : primarySymbol,
```

Comment:
Nit, since we are using `primaryIsFiat ? secondarySymbol : primarySymbol` twice, maybe we can cache it in a var called `symbol`?

Reply:
added :)